app/assets/javascripts/create.js

```diff
@@ -0,0 +1,32 @@
+$(() => {
+    createIdea()
+})
+
+var createIdea = () => {
+    $('#save-idea').click( (event) => {
+        event.preventDefault()
+        var ideaParams = {
+            title: $('#title-field').val(),
+            body:  $('#body-field').val()
+        }
+
+        $.ajax({
+            type: "POST",
+            url: "api/v1/ideas",
```

Comment:
Prefix the URL with a `/` that way you always start at the root of the application. That said, it doesn't matter in this application because we only have one page.